betfair/betfair.py

```diff
@@ -90,6 +96,45 @@ def login(self, username, password):
             raise exceptions.BetfairLoginError(response, data)
         self.session_token = data['sessionToken']
 
+    def interactive_login(self, username, password):
```

Comment:
Why make this request with `subprocess` and `curl` instead of using `requests` in python? We already have the `make_auth_request` method, which will do most of this work for you without the extra overhead of formatting commands for subprocess and parsing the results.